concreate/generator.py

```diff
@@ -19,11 +19,13 @@
 
 class Generator(object):
 
-    def __init__(self, descriptor_path, target, overrides):
-        self.descriptor = Descriptor(descriptor_path, 'image').process()
-        self.target = target
+    def __init__(self, args):
 
-        if overrides:
+        self.descriptor = Descriptor(args.descriptor, 'image').process()
+        self.target = args.target
+        self.build_tags = args.build_tags
```

Comment:
maybe this should not be part of constructor but build can take tags as an argument?